plaidml2/bridge/keras/__init__.py

```diff
@@ -1062,14 +1062,23 @@ def not_equal(lhs, rhs):
 def normalize_batch_in_training(x, gamma, beta, reduction_axes, epsilon=1e-3):
     I = x.tensor
     ndims = I.shape.ndims
```

Comment:
Since we're at the Keras level, I recommend replacing these two line with `ndims = ndim(x)`